xbmc/interfaces/builtins/GUIControlBuiltins.cpp

```diff
@@ -88,12 +89,14 @@ static int SendMessage(const std::vector<std::string>& params)
  *  \param params The parameters.
  *  \details params[0] = ID of control.
  *           params[1] = ID of subitem of control (optional).
+ *           params[2] = "absolute" to focus the absolute position instead of the relative one (optional).
  */
 static int SetFocus(const std::vector<std::string>& params)
 {
   int controlID = atol(params[0].c_str());
   int subItem = (params.size() > 1) ? atol(params[1].c_str())+1 : 0;
-  CGUIMessage msg(GUI_MSG_SETFOCUS, g_windowManager.GetFocusedWindow(), controlID, subItem);
+  int absID = (params.size() > 2) ? StringUtils::EqualsNoCase(params[2].c_str(), "absolute") : 0;
```

Comment:
can you make StringUtils::EqualsNoCase() explicitly resolve to 0 or 1, instead of implicitly relying on true == 1?